index.html

```diff
@@ -8,7 +8,178 @@
     <title>PricingPage</title>
     <link rel="preconnect" href="https://fonts.googleapis.com" />
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500;600;700&display=swap"
-      rel="stylesheet"/>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@500;600;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
+
+  <body>
+    <header>
+      <button class="btn-back">
+        <img class="img" src="img/Arrow Forward.svg" />Back
+      </button>
+      <h1 class="title">Pricing</h1>
+      <div class="subtitle">You can update or downgrage at any time</div>
+    </header>
+
+    <div class="switch-container">
+      <span class="monthly">Monthly</span>
+      <img class="img-switch" src="/img/Switch.svg" />
+      <span class="annually">Annually</span>
+      <span class="save">(save 10%)</span>
+    </div>
+
+    <main class="main">
+      <!-- block 1 -->
+      <div class="plan">
+        <div class="block-header">
+          <div class="avatar1"></div>
```

Comment:
if your element don't have content close it in one tag
```suggestion
          <div class="avatar1" />
```